time ts_a
time ts_b
int32 id_a
int32 id_b
geometry_msgs/Point dpos
float64 dyaw
geometry_msgs/Pose self_pose_a
geometry_msgs/Pose self_pose_b